### DISCUSSION TITLE
Addons when you Google Translate them through a lot of languages and then back to English ⁉️

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#7322] [Discussion]
This is why we have a localization team.

---

### 🗄️ A prosiect cabinet (Old studio layout)
Page layouts return layouts that were used before the update feature or were previously used as part of the Scratch 2.0 → 3.0 plug-in.
> [!IMPORTANT]
> This plug-in does not work on mobile devices or small windows.

> [!TIP]
> Most of the two bodies are the same size. To view multiple programs and videos at the same time, use the “More applications for applications” extension.

### ⚙️ Panel automation (Auto-hiding block palette)
If it doesn't jump, hide the button. Click the lock icon to temporarily lock it. You can use the click share feature to show or hide blocks that you haven't clicked before.
> [!TIP]
> You can adjust the position of the floating palette by simply expanding the display area. To drag a block to this location without leaving the trash can, use the other options.

### ⏹️ Turn off active play (Muted project player mode)
Press Ctrl+F to add a comma or comma to the new logo.

### 🥊 Crushed or retractable lid (Collapsing sprite properties)
No matter how much you want to do with Sprite, start with Scratch 2.0. He said that he would not be able to clean him up, but he would have to wait for him. He said to us that he had brought us all kinds of things, so that we could have our children and our children and our children, and our children, and our children, and our children, and our children.

### ❌ Make sure the spray is canceled (Sprite deletion confirmation)
You will be prompted to download the browser as part of the project.
> [!TIP]
> Tip: If you accidentally remove the focus, outfit, or sound, you can remove the feature by clicking the Edit button in the menu bar.

### 🌡️ Table showing temperature (Gamepad support)
Bluetooth transmission via USB control/game controller or Ku

### 😕 maint tabled (Resizable comment input)
Vertical space for storing Scratch 3.0 project page notes

### ➡️ Translations in the index (Jump to custom block definition)
Smart can be individually adjusted by sliding the center arrow or by clicking on the block and pressing Shift+to adjust the keys.
> [!NOTE]
> This plugin was part of the developer tools extension, but we will continue here.

### 📋 No automatic block copying (Do not automatically run duplicated blocks)
The block was torn in the first editor and an error occurred that was not related to script execution.

### 📤 Automatic uploading (Disable auto-save)
Automatically record events during editing
> [!CAUTION]
> This disables some automatic features. Keep your changes manually each time so you don't lose them.

### 💰 The best way to reduce project costs (Higher project framerate mode)
Functions can be performed at the frame rate per second (FPS), making work easier. Hold down the Alt key to switch from the default scraping format to custom frame size format (30 frames per second), and then convert the specified 30 frames per second.
> [!NOTE]
> Since changing the size of a distorted image affects how fast a particular code runs (for example, circles), most processes use higher frame sizes in programs that process that code in these various processes. Excellent

### 🪟 This area is near the Sprite window. (Search box in sprite pane)
Use the SEARCH NAME command to add a search field to the sprite window.

### 🗑️ Automatically delete named items (Do not automatically space overlapping scripts)
Linear forces are tools that you can use to protect others.

### 🔧 Change the settings to “Apply to all experts” and “Apply to this protection only.” (Switch variables between "For all sprites" and "For this sprite only")
Add an option when entering an existing variable or list. You can change this from “Apply to all experts” to “Apply to this saver only”, and the settings are saved to the cloud. A new option has also been added that allows you to quickly change the scope by right-clicking on the variable/list.

### 👜 Multiple files in one bag (Multiple rows in backpack)
The contents of the editor bag can fit up to 5 inches x 5 inches, and the double-sided display can be used as a shelf unit.

### 💦 After installation, the spray finally comes out. (Customizable new sprite position)
It changes the thoughtful atmosphere of the new show.

### 🪴 Editor: Canels Sprout (Fix sprite pane glitching)
There may be an error in purchasing a guard, it may be misuse, or the guard may be considered an expert.

---

Happy April Fools' Day! 😂

No offense to the localization/translation team. You are all doing a wonderful job!

**Comment by BroJac5246** [comment]
I have no idea what I just read but happy April Fools' Day I guess 😂

**Comment by WorldLanguages** [comment]
Table showing temperature 🤨

**Comment by mybearworld** [comment]
We should think about "Turn off active play". I really want to add a comma (I guess a comma would be fine too) to the new logo.

**Reply by Samq64** [reply]
![S,A](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/4f9687b5-cc28-45b2-a067-4b45a444bc37)
